keyboards/tkw/grandiceps/rules.mk

```diff
@@ -0,0 +1,33 @@
+# Build Options
```

Comment:
```suggestion
```suggestion
# MCU name
MCU = STM32F411

# Build Options
```

Reply:
(edited to remove duplicate suggestion markdown)